### ISSUE TITLE
May be able to make temporal loss more stable by directly computing log-prob from pre-activation function outputs

### ISSUE THREAD

**Issue by mmcdermott** [mmcdermott/EventStreamGPT#93]
E.g., the exponential pdf given rate $\lambda$ is $\lambda e^{-\lambda x}$, so if $\lambda = e^{z}$ where $z$ is the projected output of the model, then we could just directly compute $NLL = z - x*e^z$.

**Comment by mmcdermott** [comment]
Similarly, we should use https://pytorch.org/docs/stable/generated/torch.nn.GaussianNLLLoss.html for the regression targets.

**Comment by mmcdermott** [comment]
Similarly, we should normalize the predicted outputs for TTE (labs are already normalized) to have zero log-mean and unit log-variance.

**Comment by mmcdermott** [comment]
Closing as not planned due to upcoming deprecation.
